sdk/cmake/host-tools.cmake

```diff
@@ -23,6 +23,11 @@ foreach(_host_tool ${host_tools_list})
     add_dependencies(native-${_host_tool} host-tools)
 endforeach()
 
+if(USE_CLANG_CL)
+    # FIXME: Fix host tools build with clang
+    #set(USE_CLANG_CL_ARG "-DCMAKE_C_COMPILER=clang-cl;-DCMAKE_CXX_COMPILER=clang-cl")
```

Comment:
Does not build, or does not work?

Reply:
Does not build, we have some issues to iron out first, so I prepared it and commented it out so anyone could inspect them.

Reply:
That's strange, host tools build fine with macOS's clang.